docs/SDL/GetListOfPermissions/index.md

```diff
@@ -66,8 +66,10 @@ GetListOfPermissions provide  External UCS Status to HMI
 ![GetListOfPermissions](./assets/GetListOfPermissionsInformingHMI.png)
 |||
 
+|||
```

Comment:
you sure this should have format changed? check https://smartdevicelink.com/en/guides/hmi/sdl/getlistofpermissions/

Reply:
Resolved in https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/183/commits/f878b97b51a2def86fb730215446dac8ecba24c4